tools/cli/helpers/analytics.js

```diff
@@ -0,0 +1,79 @@
+import crypto from 'crypto';
+import axios from 'axios';
```

Comment:
Can we use native [fetch](https://nodejs.org/api/globals.html#fetch) rather than axios?

See also https://github.com/Automattic/jetpack/pull/34238#issuecomment-1822357122

Reply:
Great suggestion 👍 c683913475de6f0eeadb0cf38bffdeb3fdbf3bed